packages/experimental-nextjs-app-support/README.md

```diff
@@ -204,7 +204,7 @@ That way, if you repeat the query in your Client Component using `useSuspenseQue
 
 #### Usage with `useReadQuery`.
 
-You can also use this approach in combination with `useReadQuery` in Client Components. Use the render prop approach to get a `QueryRef` that you can pass to your Client Component:
+Just like using `useBackgroundQuery` and `useReadQuery` in place of `useSuspenseQuery` (see [the suspense documentation](https://www.apollographql.com/docs/react/data/suspense#avoiding-request-waterfalls)), you can also use `PreloadQuery` in combination with `useReadQuery` in Client Components. Use the render prop notation to get a `QueryRef` that you can pass to your Client Component:
```

Comment:
```suggestion
Just like using [`useBackgroundQuery`](https://www.apollographql.com/docs/react/data/suspense#avoiding-request-waterfalls) and `useReadQuery` in place of `useSuspenseQuery`, you can also use `PreloadQuery` in combination with `useReadQuery` in Client Components. Use the render prop notation to get a `QueryRef` that you can pass to your Client Component:
```

Would it make more sense to move this link to `useBackgroundQuery`? I love the link to the documentation, but it feels a touch awkward to interrupt the sentence with the "see x" statement.

Reply:
Since I'm not linking to the `uBQ` docs, but to the "avoiding waterfalls" section as a generally related concepts, I'd probably keep it separated.

Reply:
```suggestion
Just like using `useBackgroundQuery` with `useReadQuery` in place of `useSuspenseQuery` [to avoid request waterfalls](https://www.apollographql.com/docs/react/data/suspense#avoiding-request-waterfalls), you can also use `PreloadQuery` in a similar fashion in combination with `useReadQuery` in Client Components. Use the render prop notation to get a `QueryRef` that you can pass to your Client Component:
```

But I believe this might work.